product_docs/docs/pgd/5/reference/routing.mdx

```diff
@@ -101,6 +101,6 @@ bdr.routing_leadership_transfer(node_group_name text,
 |--------------------|----------|----------|---------------------------------------------------------------------------------------------|
 | `node_group_name`  | text     |          | Name of group where the leadership transfer is requested.                                   |
 | `leader_name`      | text     |          | Name of node that will become write leader.                                                 |
-| `transfer_method`  | text     | 'strict' | Type of the transfer, it can be "fast" or the default "strict" that checks the maximum lag. |
-| `transfer_timeout` | interval | '10s'    | Timeout of the leadership transfer, default is 10 seconds.                                  |
+| `transfer_method`  | text     | 'strict' | Type of the transfer. It can be 'fast' or the default, 'strict', which checks the maximum lag. |
```

Comment:
```suggestion
| `transfer_method`  | text     | `'strict'` | Type of the transfer. It can be `'fast'` or the default, `'strict'`, which checks the maximum lag. |
```